gnomad_qc/v4/annotations/generate_variant_qc_annotations.py

```diff
@@ -92,7 +99,14 @@ def get_variant_qc_annotation_resources(
     )
     run_vep = PipelineStepResourceCollection(
         "--run-vep",
-        output_resources={"vep_ht": get_vep(test=test)},
+        output_resources={
+            "vep_ht": get_vep(test=True)
+        },  # TODO: I found "test" is a boolean here.
```

Comment:
I'm not sure what you mean, test should be a boolean, and we don't want it to always be True

Reply:
ah, sorry, I misunderstood.